canonical-web-and-design/maas-ui
# fix(ui): fix instances of tooltip icons not using buttons

## Done

- Used `TooltipButton` component wherever we had an icon show a tooltip on hover
- Slightly updated `TooltipButton` prop shape so that `iconName` is exposed at the top-level

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Navigate around the app and check that every instance of an icon tooltip is attached to a button

## Fixes

Fixes canonical-web-and-design/app-tribe#869
Fixes canonical-web-and-design/app-tribe#894
